public/js/main.js

```diff
@@ -12,8 +12,19 @@ var hideMessages = function hideMessages(){
   }, 1000);
 }
 
+var updateGoogleLink = function updateGoogleLink(){
+  var link = $('.js-jtinder li').eq(0).attr('data-google-link');
+  $('.js-google-link').attr('href', link);
+}
+
 function loadNewPerson() {
   $('.js-extra-people li:first-child').appendTo('.js-jtinder ul');
+
+  console.log( $('.js-jtinder ul').text() );
+  console.log( $('.js-jtinder li').eq(1).text() );
```

Comment:
Probably best to get rid of these `console.log` statements, I'm not sure if there are any "modern" browsers that are missing the `console` object but best to be safe